pkg/systemd/logs.js

```diff
@@ -288,6 +380,391 @@ $(function() {
+            pi.append(c1);
+            for (i = 0; i < problem_info_1.length; i++) {
+                elem = problem_info_1[i];
+                if (elem in args){
+                    val = args[elem][2];
+                    c1.append(
+                        $('<tr>').append(
+                            $('<td>').css('text-align', 'right').text(elem),
+                            $('<td>').css('text-align', 'left')
+                                .text(String(val))));
+                }
+            }
+
+            // Render second column of problem info
+            var c2 = $('<div>').css('display', 'inline-block')
+                               .css('vertical-align', 'top');
+            pi.append(c2);
+            for (i = 0; i < problem_info_2.length; i++) {
+                elem = problem_info_2[i];
+                if (elem in args){
+                    val = args[elem][2];
+                    c2.append(
+                        $('<tr>').append(
+                            $('<td>').css('text-align', 'right').text(elem),
+                            $('<td>').css('text-align', 'left')
+                                .text(String(val))));
+                }
+            }
+
+            // Render problem details
+            var problem_details_elems = Object.keys(problem_render_callbacks);
+            $.each(problem_details_elems, function(i, key) {
+                if (key in args){
+                    val = problem_render_callbacks[key](args[key]);
+                    pd.append(
+                        $('<div class="panel panel-default">')
+                            .css("border-width", "0px 0px 2px 0px")
+                            .append(
+                          $('<div class="panel-heading problem-panel">')
+                            .attr('data-toggle', 'collapse')
+                            .attr('data-target', '#' + key)
+                            .attr('data-parent', '#accordion-markup')
+                            .append($('<h4 class="panel-title">')
+                              .append($('<a class="accordion-toggle">')
+                                .text(key))),
+                          $('<div class="panel-collapse collapse">')
+                            .attr('id', key)
+                            .append(
+                              $('<div class="panel-body">')
+                                    .html(val))));
+                }
+            });
+        });
+    }
+
+    function render_table_eq(orig) {
+        return render_table(orig, '=');
+    }
+
+    function render_table_co(orig) {
+        return render_table(orig, ':');
+    }
+
+    function render_table(orig, delimiter) {
+        var lines = orig[2].split('\n');
+        var result;
+
+        for (var i = 0; i < lines.length - 1; i++) {
+            var line = lines[i].split(delimiter);
+            result += '<tr> <td style="test-align: right">' + line[0];
```

Comment:
We can not use inline styles like this because of the Content-Security-Policy.  Just use `class="text-right"`.